packages/photon/package-lock.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
can drop the package lock entirely and add a .npmrc that disallows package locks

Reply:
28b76fb770b98ac19fc500a54acdc78cd136835c